litmus-portal/frontend/src/views/CreateWorkflow/CustomWorkflow/CreateWorkflow/index.tsx

```diff
@@ -224,7 +244,7 @@ const CreateWorkflow: React.FC<VerifyCommitProps> = ({ gotoStep }) => {
               styles={{
```

Comment:
can This be added to styles.ts?

Reply:
@vanshBhatia-A4k9 As Custom InputField can take styles as props, that's why we are providing as props.